ksqldb-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -436,6 +438,24 @@ protected Void visitListTables(final ListTables node, final Integer context) {
       return null;
     }
 
+    @Override
+    protected Void visitDescribeStreams(final DescribeStreams node, final Integer context) {
+      builder.append("DESCRIBE STREAMS");
+      if (node.getShowExtended()) {
+        visitExtended();
+      }
```

Comment:
I think this will create a `DESCRIBE STREAMS EXTENDED`. Could you add a test to the SqlFormatterTest to validate this?